Makefile

```diff
@@ -110,6 +110,14 @@ push-image:
 	$(CTR_CMD) push $(IMAGE_REPO):$(IMAGE_TAG)
 .PHONY: push-image
 
+multi-arch-image-base:	
+	docker pull --platform=linux/s390x quay.io/sustainable_computing_io/kepler_base:latest-s390x; \
```

Comment:
can you replace `docker` with `${CTR_CMD}`? We usually run `podman` rather than docker during development.

Reply:
Done